test/open_dkim.bats

```diff
@@ -21,7 +21,7 @@ function setup_file
     --name "${CONTAINER_NAME}" \
 		--cap-add=SYS_PTRACE \
 		-v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
-		-v "${CDIR}/test/test-files":/tmp/docker-mailserver-test:ro \
+		-v "$(pwd)/test/test-files":/tmp/docker-mailserver-test:ro \
```

Comment:
```suggestion
		-v "$PWD/test/test-files":/tmp/docker-mailserver-test:ro \
```

Reply:
We use $(pwd) in a ton of other tests. What's the reason to use $PWD here instead?

Reply:
Oh no, it was for me more practical then calling a binary
But if tests already use it then its perfect that way ;)

Reply:
Roger that -- Within the tests/* directory I see 60 occurrences of `$(pwd)` :)